### PR TITLE
NAS-128385 / 24.10 / Port CPU widget to new dashboard

### PR DESCRIPTION
**Pull request by bvasilenko**
**Testing**

Verify widget at the new dashboard

### CODE REVIEW

File: src/app/pages/dashboard/widgets/cpu/widget-cpu/widget-cpu.component.ts [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+import { map } from 'rxjs/operators';
+import { AllCpusUpdate } from 'app/interfaces/reporting.interface';
+import { GaugeConfig, GaugeData } from 'app/modules/charts/components/view-chart-gauge/view-chart-gauge.component';
+import { WidgetResourcesService } from 'app/pages/dashboard/services/widget-resources.service';
+import { SlotSize } from 'app/pages/dashboard/types/widget.interface';
+import { ThemeService } from 'app/services/theme/theme.service';
+import { AppState } from 'app/store';
+import { waitForSystemInfo } from 'app/store/system-info/system-info.selectors';
+
+interface CpuParams {
```
Comment (RehanY147) [inline src/app/pages/dashboard/widgets/cpu/widget-cpu/widget-cpu.component.ts:18]: Interface shouldn't be here.

**Review by RehanY147 — APPROVED** [review]
Looks good. The CPU widget is showing up on the new dashboard.

**Comment by AlexKarpov98** [comment]
@bvasilenko - please update https://github.com/truenas/webui/pull/10009/files#diff-30b70af9b0d2da808112496b0366f5db251fefc62238f754f8bb8d4637cffeceR111 - to include this widget as well.

**Review by AlexKarpov98 — COMMENTED** [review]
@bvasilenko - please update https://github.com/truenas/webui/pull/10009/files#diff-30b70af9b0d2da808112496b0366f5db251fefc62238f754f8bb8d4637cffeceR111 - to include this widget as well.

**Pull request merged**